### ISSUE TITLE
Linux Install

### ISSUE THREAD

**Issue by xfacta** [ArduPilot/MethodicConfigurator#1]
Great work!
Not a real issue but just suggestions:

Please add an instruction in readme.md for the linux user to run
`chmod +x install_linux.sh`
before running the install script

A launcher for Ubuntu and probably other debian-based distros could be created and saved as "`MethodicConfigurator.desktop`" in `/home/$USER/.local/share/applications`
The linux install script might even be able to do this if you can detect the linux distro to ensure it's debian-based.

```
[Desktop Entry]
Name=Methodic Configurator
Comment=ArduPilot Methodic Configurator
Exec=python3 /home/$USER/MethodicConfigurator/MethodicConfigurator/ardupilot_methodic_configurator.py
Icon=/home/$USER/MethodicConfigurator/MethodicConfigurator/ArduPilot_icon.png
Terminal=true
Type=Application
Categories=Development;
Keywords=ardupilot;arducopter;drone;copter;
```

The .desktop file must have the users actual home path in it and cannot use $HOME or $USER, so the alternatives are:
a) end-user would have to modify the .desktop file and add in their own correct path
b) the install script would have to populate it
c) other options get very complex, not worth discussing I think

**Comment by amilcarlucas** [comment]
Thanks @xfacta usually I do this is a complicated way using Ansible and jinja template files.
I guess this time I will just use bash :)

**Comment by amilcarlucas** [comment]
Fixed by fe55993
